docker/dockerfiles/Dockerfile.default-gpu

```diff
@@ -92,6 +91,7 @@ RUN python3 -m pip install -U \
       scikit-image \
       nltk
 
+RUN pip2 install spacy
```

Comment:
Would you mind updating the cpu docker image to have the same lines?  Want to make sure the two images stay in sync for the prefix.